### ISSUE TITLE
Example datasets for download don't exist?

### ISSUE THREAD

**Issue by XapaJIaMnu** [jelmervdl/empty-train#3]
```bash
% mtdata get -l ara-eng -tr OPUS-elrc_2922-v1-ara-eng --compress -o data                                                                                                                                              :(
2022-06-22 09:41:26 main.get_data:32 WARNING:: Args are ignored: {'verbose': False, 'reindex': False, 'task': 'get'}
2022-06-22 09:41:26 __init__.get_instance:48 INFO:: Loading index from cache /home/dheart/.mtdata/mtdata.index.0.3.4.pkl
Traceback (most recent call last):
  File "/usr/bin/mtdata", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3.10/site-packages/mtdata/__main__.py", line 9, in main
    main.main()
  File "/usr/lib/python3.10/site-packages/mtdata/main.py", line 215, in main
    get_data(**vars(args))
  File "/usr/lib/python3.10/site-packages/mtdata/main.py", line 35, in get_data
    dataset = Dataset.prepare(
  File "/usr/lib/python3.10/site-packages/mtdata/data.py", line 67, in prepare
    all_entries = cls.resolve_entries(all_dids)
  File "/usr/lib/python3.10/site-packages/mtdata/data.py", line 53, in resolve_entries
    raise Exception(f'Could not find {did}; try "mtdata list | grep -i {did}" to locate it')
Exception: Could not find OPUS-elrc_2922-v1-ara-eng; try "mtdata list | grep -i OPUS-elrc_2922-v1-ara-eng" to locate it
```
Same for the other one. Were all elrc elitr:

```bash
% mtdata list | grep OPUS-elrc                                                                                                                                                                                        :(
2022-06-22 09:42:39 __init__.get_instance:48 INFO:: Loading index from cache /home/dheart/.mtdata/mtdata.index.0.3.4.pkl
2022-06-22 09:42:40 main.list_data:26 INFO:: Total 182876 entries
```

Were they removed from mtdata?

```bash
% mtdata --version                                                                                                                                                                                                    
mtdata 0.3.4 /usr/lib/python3.10/site-packages/mtdata

% mtdata list | grep ELRC | grep ara-eng
2022-06-22 09:49:45 __init__.get_instance:48 INFO:: Loading index from cache /home/dheart/.mtdata/mtdata.index.0.3.4.pkl
ELRC-wikipedia_health-1-ara-eng https://elrc-share.eu/repository/download/f9019bde8de811ea913100155d026706d2e1ada233b5473f826a75d404c09c5f/     en-ar.tmx
2022-06-22 09:49:46 main.list_data:26 INFO:: Total 182876 entries
```

**Comment by jelmervdl** [comment]
The elitr is a copy/paste mistake by me. I'll push a fix.

The ELRC ara-eng one might be very recent?
```
% mtdata --version
mtdata 0.3.5 /usr/local/lib/python3.9/site-packages/mtdata

% mtdata list | grep ELRC | grep ara-eng
2022-06-22 10:57:54 __init__.get_instance:48 INFO:: Loading index from cache /Users/jelmer/.mtdata/mtdata.index.0.3.5.pkl
2022-06-22 10:57:56 main.list_data:27 INFO:: Total 131301 entries
OPUS-elrc_2922-v1-ara-eng	https://object.pouta.csc.fi/OPUS-ELRC_2922/v1/moses/ar-en.txt.zip	*.ar,*.en
ELRC-wikipedia_health-1-ara-eng	https://elrc-share.eu/repository/download/f9019bde8de811ea913100155d026706d2e1ada233b5473f826a75d404c09c5f/	en-ar.tmx

**Comment by jelmervdl** [comment]
Fixed in 305cc0fd53bbe6b50d1e800da9a3511cedab99f8
